app/views/player.py

```diff
@@ -96,6 +102,7 @@ def post(self, request):
 
 @method_decorator(login_required, name='dispatch')
```

Comment:
you can probably delete login required here, as player required probably requires login